test/jdk/tools/jlink/plugins/VendorInfoPluginsTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This should be "2019, 2014,"

Also you can bump the date on the module-info.java.